src/utils/nextPurchaseDay.js

```diff
@@ -0,0 +1,25 @@
+/***
+ * Firts we verified by specificity and
+ * we return the string that is used for the
+ * aria-label and to add the proper style
+ */
+
+export const nextPurchaseDay = (
+  daysUntilNextPurchase,
+  lastPurchaseDate,
+  numberOfPurchases,
+) => {
+  const nextEstimatedPurchase = 2 * daysUntilNextPurchase;
+  if (numberOfPurchases === 1 || lastPurchaseDate === nextEstimatedPurchase) {
+    return 'inactive';
+  }
+  if (daysUntilNextPurchase < 7) {
+    return 'soon';
+  }
+  if (daysUntilNextPurchase >= 7 || daysUntilNextPurchase <= 30) {
+    return 'kindOfSoon';
+  }
```

Comment:
I think the correct comparison would be done with `'&&'` as we want the value to be between the range of 7 and 30 days. Having this `'||'` a value greater than 30 meets the condition and would be considered as `kindOfSoon`.

```suggestion
  if (daysUntilNextPurchase >= 7 && daysUntilNextPurchase <= 30) {
    return 'kindOfSoon';
  }
```

Reply:
Thanks for the recommendation, it looks like you are absolutely right. We are making this change 😄